.github/workflows/pull-request.yml

```
@@ -0,0 +1,49 @@
+name: Pull Request Creation
+
+on:
+  push:
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)